### PR TITLE
Creation of language Spanish

### PR DESCRIPTION
**Pull request by roliverosc-zz**
Hello @enen92 I have allowed myself to create the Spanish translation for your screensaver. I hope you like it and that you approve it.

Regards.

### CODE REVIEW

File: resources/language/resource.language.es_es/strings.po [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+msgid "Kaster"
+msgstr ""
+
+msgctxt "#32001"
+msgid "Photo by"
+msgstr "Fotografía de"
+
+msgctxt "#32002"
+msgid "General"
+msgstr ""
```
Comment (enen92) [inline resources/language/resource.language.es_es/strings.po]: Missing

File: addon.xml [hunk 2]
```diff
@@ -12,11 +12,13 @@
 		<forum>https://forum.kodi.tv/showthread.php?tid=319709</forum>
 		<summary lang="en_GB">A chromecast-like screensaver for Kodi</summary>
 		<summary lang="pt_PT">Um screensaver idêntico ao screensaver original chromecast</summary>
+		<summary lang="es_SP">Un protector de pantalla tipo Chromecast para Kodi</summary>
 		<description lang="en_GB">Display beautiful pictures originally from the chromecast screensaver. You can also display your own photos along with its respective information.</description>
 		<description lang="pt_PT">Apresenta imagens lindíssimas originalmente pertencentes ao screensaver do chromecast. Pode também apresentar as suas próprias imagens conjuntamente com a respectiva informação.</description>
+		<description lang="es_SP">Muestra bellas imágenes como las del protector de pantalla de chromecast. También puede mostrar sus propias fotos junto con su respectiva información.</description>
```
Comment (enen92) [inline addon.xml]: This should be es_es

File: addon.xml [hunk 3]
```diff
@@ -12,11 +12,13 @@
 		<forum>https://forum.kodi.tv/showthread.php?tid=319709</forum>
 		<summary lang="en_GB">A chromecast-like screensaver for Kodi</summary>
 		<summary lang="pt_PT">Um screensaver idêntico ao screensaver original chromecast</summary>
+		<summary lang="es_SP">Un protector de pantalla tipo Chromecast para Kodi</summary>
```
Comment (enen92) [inline addon.xml]: This should be es_es

File: addon.xml [hunk 4]
```diff
@@ -12,11 +12,13 @@
...
 		<description lang="en_GB">Display beautiful pictures originally from the chromecast screensaver. You can also display your own photos along with its respective information.</description>
 		<description lang="pt_PT">Apresenta imagens lindíssimas originalmente pertencentes ao screensaver do chromecast. Pode também apresentar as suas próprias imagens conjuntamente com a respectiva informação.</description>
+		<description lang="es_SP">Muestra bellas imágenes como las del protector de pantalla de chromecast. También puede mostrar sus propias fotos junto con su respectiva información.</description>
 		<news>v1.0.2 [CR]
 			- Hide logos. [CR]
-			-animations
+			- Animations
```
Comment (enen92) [inline addon.xml]: Please drop this change as it refers to changes between versions. It will be changed in the next update

File: addon.xml (hunk 3 above)
Comment (enen92) [reply addon.xml]: es_ES

File: addon.xml (hunk 2 above)
Comment (enen92) [reply addon.xml]: es_ES

File: resources/language/resource.language.es_es/strings.po [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+# Addon id: screensaver.kaster
+# Addon Provider: enen92
+msgid ""
+msgstr ""
+"Project-Id-Version: XBMC Addons\n"
+"Report-Msgid-Bugs-To: alanwww1@xbmc.org\n"
+"POT-Creation-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
+"Last-Translator: Kodi Translation Team\n"
+"Language-Team: English (http://www.transifex.com/projects/p/xbmc-addons/language/en/)\n"
```
Comment (enen92) [inline resources/language/resource.language.es_es/strings.po]: Please adapt this entry

**Review by enen92 — CHANGES_REQUESTED** [review]
Thanks. Can you make the requested changes?

File: addon.xml (hunk 3 above)
Comment (roliverosc-zz) [reply addon.xml]: Sorry, I've already changed it.

**Comment by roliverosc-zz** [comment]
I've already changed it

**Comment by roliverosc-zz** [comment]
Already I've changed everything.

**Review by enen92 — APPROVED** [review]

**Pull request merged**